src/main/minio.js

```diff
@@ -1631,18 +1627,15 @@ export class Client {
   // Calls implemented below are related to multipart.
 
   // Initiate a new multipart upload.
-  initiateNewMultipartUpload(bucketName, objectName, contentType, cb) {
+  initiateNewMultipartUpload(bucketName, objectName, metaData, cb) {
     if (!isValidBucketName(bucketName)) {
       throw new errors.InvalidBucketNameError('Invalid bucket name: ' + bucketName)
     }
     if (!isValidObjectName(objectName)) {
       throw new errors.InvalidObjectNameError(`Invalid object name: ${objectName}`)
     }
-    if (!isString(contentType)) {
-      throw new TypeError('contentType should be of type "string"')
-    }
```

Comment:
you can add `if (!isObject(metaData)) {...}`

Reply:
Should be good now.